src/sentry/incidents/action_handlers.py

```diff
@@ -106,7 +107,7 @@ def resolve(self, metric_value: int | float, new_status: IncidentStatus):
 
     def email_users(self, trigger_status: TriggerStatus, incident_status: IncidentStatus) -> None:
         for user_id, email in self.get_targets():
-            user = User.objects.get_from_cache(id=user_id)
+            user = user_service.get_user(user_id=user_id)
```

Comment:
This is going to do N RPC calls. Should we refactor this to use `user_service.get_many()` instead?

Reply:
@markstory Done!